symbolic-debuginfo/src/dwarf.rs

```diff
@@ -1320,6 +1344,31 @@ impl<'s> Iterator for DwarfUnitFileIterator<'s> {
     }
 }
 
+fn resolve_byte_name<'s>(bcsymbolmap: Option<&BcSymbolMap<'s>>, s: &'s [u8]) -> &'s [u8] {
+    if let Some(bcsymbolmap) = bcsymbolmap {
+        std::str::from_utf8(s)
+            .map(|s| bcsymbolmap.resolve(s))
```

Comment:
What would you think about adding a binary version of `resolve`; or even changing it to work with `AsRef<[u8]>`?